task/control-plane-agent/src/update/rot.rs

```diff
@@ -235,11 +241,17 @@ impl ComponentUpdater for RotUpdate {
                 match self.task.abort_update() {
                     // Aborting an update that hasn't started yet is fine;
                     // either way our caller is clear to start a new update.
-                    Ok(()) | Err(SprotError::UpdateNotStarted) => {
+                    Ok(())
+                    | Err(SprotError::Update(
+                        drv_sprot_api::UpdateError::UpdateNotStarted,
```

Comment:
This is kind of annoying, maybe the abort operation on the update API should return Ok if the update is not started? I believe this would also make it idempotent.

Reply:
Fixed